content/en/blog/_posts/2018-07-12-resizing-persistent-volumes-using-kubernetes.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: blog
+title: 'Resizing Persistent Volumes using Kubernetes'
+date: 2018-07-12
+---
+
+**Author**: Hemant Kumar (Red Hat)
+
+**Editor’s note: this post is part of a [series of in-depth articles](https://kubernetes.io/blog/2018/06/27/kubernetes-1.11-release-announcement/) on what’s new in Kubernetes 1.11**
+
+In Kubernetes v1.11 the persistent volume expansion feature is being promoted to beta. This feature allows users to easily resize an existing volume by editing the PersistentVolumeClaim object. Users no longer have to manually interact with the storage backend or delete and recreate PV and PVC objects to increase the size of a volume. Shrinking persistent volumes is not supported.
```

Comment:
PersistentVolumeClaim should be in " ` " quotes (API object).

Reply:
i would add `(PVC)` after `PersistentVolumeClaim` to indicate that the rest of the doc uses this abbreviation.